### PR TITLE
Add in ERA5 as option for ISCE_INSAR

### PR DESCRIPTION
**Pull request by jacquelynsmale**

### CODE REVIEW

File: job_spec/INSAR_ISCE.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ INSAR_ISCE:
           - HRRR
           - HRES
           - GMAO
+          - ERA5
```
Comment (jhkennedy) [inline job_spec/INSAR_ISCE.yml:40]: Might actually be good to include *all* of these:
https://github.com/dbekaert/RAiDER/blob/develop/tools/RAiDER/models/allowed.py

Comment (jacquelynsmale) [reply job_spec/INSAR_ISCE.yml:40]: @jhkennedy Sweet. Added!

File: job_spec/INSAR_ISCE.yml [hunk 2]
```diff
@@ -37,6 +37,10 @@ INSAR_ISCE:
...
+          - NCMR
+          - ERA5
+          - ERA5T
```
Comment (jhkennedy) [inline job_spec/INSAR_ISCE.yml:43]:
```suggestion
          - ERA5
          - ERA5T
          - GMAO
          - HRES
          - HRRR
          - NCMR
```

I think ERA5 is duplicated here. Also, might be nice to alphabetize these.

File: job_spec/INSAR_ISCE_TEST.yml [hunk 3]
```diff
@@ -37,6 +37,9 @@ INSAR_ISCE_TEST:
           - HRRR
           - HRES
           - GMAO
+          - NCMR
+          - ERA5
+          - ERA5T
```
Comment (jhkennedy) [inline job_spec/INSAR_ISCE_TEST.yml:42]:
```suggestion
          - ERA5
          - ERA5T
          - GMAO
          - HRES
          - HRRR
          - NCMR
```
Same here

**Review by jhkennedy — APPROVED** [review]
Looks good to me!

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,8 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [3.0.0]
+## [3.0.0]
```
Comment (jhkennedy) [inline CHANGELOG.md:7]:
```suggestion
## [3.0.0]
```
totally inconsequential

**Review by jhkennedy — APPROVED** [review]

**Comment by jacquelynsmale** [comment]
@jhkennedy the approval request that never ends 😅

**Review by jhkennedy — APPROVED** [review]
This is the review that never ends. It goes on and on my friends... 🤣

**Comment by jhkennedy** [comment]
part of #1416

**Pull request merged**